apps/web/src/routes/_content/resources/newsletters/index.tsx

```diff
@@ -16,49 +19,134 @@ export const Route = createFileRoute('/_content/resources/newsletters/')({
 });
 
 function Newsletter() {
-  const [searchTerm, setSearchTerm] = useState('');
   const { t, i18n } = useTranslation();
+  const isMobile = useSmaller('md');
+
   const { data: newsletters, isFetched } = trpc.content.getNewsletters.useQuery(
-    {
-      language: i18n.language,
-    },
-    {
-      staleTime: 300_000, // 5 minutes
-    },
+    {},
+    { staleTime: 300_000 },
   );
 
-  const filteredNewsletters =
-    newsletters?.filter((newsletter) =>
-      newsletter.title?.toLowerCase().includes(searchTerm.toLowerCase()),
+  const localNewsletters =
+    newsletters?.filter(
+      (newsletter) => newsletter.language === i18n.language,
     ) ?? [];
+
+  const englishNewsletters =
+    newsletters?.filter((newsletter) => newsletter.language === 'en') ?? [];
+
+  const [showLocalOnly, setShowLocalOnly] = useState(false);
+
+  const handleSwitchChange = (checked: boolean) => {
+    setShowLocalOnly(checked);
+  };
+
+  const sortedNewsletters = (
```

Comment:
might be better to create a shallow copy of localNewsletters/newsletters instead of sorting them directly (eg: [...localNewsletters]) -> would preserve the original array